src/test/java/com/example/checkstylereviewdogsample/utils/SampleUtilsTest.java

```diff
@@ -0,0 +1,10 @@
+package com.example.checkstylereviewdogsample.utils;
+
+import org.junit.jupiter.api.Test;
+
+public class SampleUtilsTest {
+    @Test
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 4, expected level should be 2.